include/RED4ext/ResourceDepot.hpp

```diff
@@ -39,7 +40,10 @@ RED4EXT_ASSERT_OFFSET(ArchiveGroup, basePath, 0x10);
 
 struct ResourceDepot
 {
+    virtual Memory::IAllocator GetAllocator() = 0;
     virtual ~ResourceDepot() = 0;
+    virtual void* RequestResource(void* aHandle, ResourcePath aPath, int32_t* aContext) = 0;
+    virtual bool IsResourceExists(ResourcePath aPath) = 0;
```

Comment:
I would suggest that these two to be `Request` and `Exists`,  since it is in a depot.

Reply:
Depot also operates with archives. Potentially there can be `IsArchiveExists()`, `OpenArchive()`, etc.

Reply:
Then what about `DoesResourceExist` or `IsResourcePresent`? I find it bit odd to read `IsResourceExists`.

Reply:
Can be just `ResourceExists()`.

Reply:
That is also good.

Reply:
Done.